triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/TriggerDao.scala

```diff
@@ -41,11 +43,9 @@ object TriggerDao {
   def initialize(implicit log: LogHandler): ConnectionIO[Unit] = {
     val createTriggerTable: Fragment = sql"""
         create table running_triggers(
-          trigger_id uuid primary key,
+          trigger_instance uuid primary key,
           party_token text not null,
-          package_id text not null,
-          module_name text not null,
-          trigger_name text not null
+          full_trigger_name text not null
```

Comment:
Please document the format of the name if you want to store it as a single string.

Reply:
Do you just mean a comment in this code or somewhere else?

Reply:
Just here in the code is totally fine.